lantern/flask/flask_SQLalchemy/practice/application/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.8
+COPY . /app
+WORKDIR /app
+RUN pip install --upgrade pip
+RUN pip install -r requirements.txt
+CMD python wsgi.py
```

Comment:
you should read about wsgi, and for what purpose it uses
Please rename this file, as you mislead others with this name